intrinsic_cal/src/rail_cal.cpp

```diff
@@ -139,11 +146,11 @@ RailCalService::RailCalService(ros::NodeHandle nh)
   camera_parameters_.position[1] = pose.y;
   camera_parameters_.position[2] = pose.z;
 
-  camera_parameters_.focal_length_x = 2785.0;
-  camera_parameters_.focal_length_y = 2781.0;
-  camera_parameters_.center_x = 963.505;
-  camera_parameters_.center_y = 575.686;
-  
+  camera_parameters_.focal_length_x = 530.0;
```

Comment:
so you want the default to be for the kinect, not a basler with a 6mm lens? ICs now come from camera server anyway, so why change it?